.github/workflows/pr-labels.yml

```diff
@@ -1,50 +0,0 @@
-name: pr-labels
-on:
-  pull_request:
-    types: [opened, labeled, unlabeled, synchronize]
-jobs:
-  analyze:
-    if: github.repository == 'vitessio/vitess'
-    name: analyze_pr_labels
-    runs-on: ubuntu-latest
-    steps:
-    - name: Tune the OS
-      run: |
-        echo '1024 65535' | sudo tee -a /proc/sys/net/ipv4/ip_local_port_range
-
-    # TEMPORARY WHILE GITHUB FIXES THIS https://github.com/actions/virtual-environments/issues/3185
-    - name: Add the current IP address, long hostname and short hostname record to /etc/hosts file
-      run: |
-        echo -e "$(ip addr show eth0 | grep "inet\b" | awk '{print $2}' | cut -d/ -f1)\t$(hostname -f) $(hostname -s)" | sudo tee -a /etc/hosts
-    # DON'T FORGET TO REMOVE CODE ABOVE WHEN ISSUE IS ADRESSED!
-
-    - name: analyze labels
-      env:
-        PR_NUMBER: ${{ github.event.pull_request.number }}
-      run: |
-        if [ -z "$PR_NUMBER" ] ; then
-          exit 0
-        fi
-        LABELS_JSON="/tmp/labels.json"
-
-        # Get labels for this pull request
-        curl -s \
-          -H 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
-          -H "Accept: application/vnd.github.v3+json" \
-          -H "Content-type: application/json" \
-          "https://api.github.com/repos/${GITHUB_REPOSITORY}/issues/${PR_NUMBER}/labels" \
-          > "$LABELS_JSON"
-
-        if ! cat ${LABELS_JSON} | jq -r '.[].name ' | grep -q 'Component:' ; then
-          echo "Expecting PR to have label 'Component: ...'"
-          exit 1
-        fi
-        if ! cat ${LABELS_JSON} | jq -r '.[].name ' | grep -q 'Type:' ; then
-          echo "Expecting PR to have label 'Type: ...'"
-          exit 1
```

Comment:
Let us not delete this file. Instead, we can just remove the `release notes` label checking from here.
It is still important to set `Component` and `Type` so that PRs appear in the right section of the changelog.

Reply:
My bad, I added it back in the new workflow file to avoid having two workflows that do (almost) the same job